src/components/SingleArticle.jsx

```diff
@@ -28,6 +29,8 @@ const SingleArticle = () => {
  <p>{selectedArticle.body}</p></section>}
  <VoteForArticle article_id={article_id} voteCount={voteCount} setVoteCount={setVoteCount} hasVoted={hasVoted} setHasVoted={setHasVoted} setError={setError}/>
  <CommentsForArticle article_id={article_id}/>
+ <p></p>
+ <NewComment article_id={article_id} user={user}/>
```

Comment:
Does this want to be below all the other comments? Have a look at other websites eg youtube/facebook etc., how do they render their add comment components?